modules/S115_usermode_emulator.sh

```diff
@@ -103,11 +104,27 @@ version_detection() {
   echo
 }
 
+detect_root_dir() {
+  INTERPRETER_FULL_PATH=$(find "$EMULATION_PATH_BASE" -type f -executable -exec file {} \; | grep "ELF" | grep "interpreter" | sed s/.*interpreter\ // | sed s/,\ .*$// | sort -u 2>/dev/null)
+  if [[ $(echo "$INTERPRETER_FULL_PATH" | wc -l) -gt 0 ]]; then
+    # now we have a result like this "/lib/ld-uClibc.so.0"
+    INTERPRETER=$(echo "$INTERPRETER_FULL_PATH" | sed -e 's/\//\\\//g')
+    #EMULATION_PATH=$(find "$EMULATION_PATH" -wholename "*$INTERPRETER_FULL_PATH" | sort -u | sed -e 's/$INTERPRETER//')
+    EMULATION_PATH=$(find "$EMULATION_PATH_BASE" -wholename "*$INTERPRETER_FULL_PATH" | sort -u)
+    #EMULATION_PATH=$(echo "$EMULATION_PATH" | sed -e s/"$INTERPRETER"//)
+    EMULATION_PATH=$(echo "${EMULATION_PATH//$INTERPRETER/}")
```

Comment:
Shellcheck error
```
In ./modules/S115_usermode_emulator.sh line 115:
    EMULATION_PATH=$(echo "${EMULATION_PATH//$INTERPRETER/}")
Error:                    ^-- SC2116: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.
```